backend/plugins/mongolian_api/src/modules/ebarimt/afterMutations.ts

```
@@ -0,0 +1,178 @@
+            });
+          }
+        }
+      } else {
+        try {
+          const { putData, innerData } = await models.PutResponses.putData(
+            ebarimtData,
+            config,
+            user,
+          );
+
+          putData && ebarimtResponses.push(putData);
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)